OBSDOCS-674: Split LokiStack CR into own modules, move object storage docs (#68952)
openshift/openshift-docs: modules/logging-loki-cli-install.adoc

// Module is included in the following assemblies:
// logging/log_storage/installing-log-storage.adoc

:_mod-docs-content-type: PROCEDURE
[id="logging-loki-cli-install_{context}"]
= Installing {loki-op} by using the CLI

To install and configure logging on your {product-title} cluster, additional Operators must be installed. This can be done from the {product-title} CLI.

{Product-title} Operators use custom resources (CR) to manage applications and their components. High-level configuration and settings are provided by the user within a CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the Operator’s logic. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs, which are then used to generate CRs.

.Prerequisites

* You have administrator permissions.

> * You have installed the {oc-first}.

Reviewer: ```suggestion
* You installed the {oc-first}.
```

> * You have access to a supported object store. For example: AWS S3, Google Cloud Storage, Azure, Swift, Minio, or OpenShift Data Foundation.

Reviewer: ```suggestion
* You have access to a supported object store. For example: AWS S3, Google Cloud Storage, Azure, Swift, Minio, or {rh-storage}.
```

> <2> Specify `stable`, or `stable-5.<y>` as the channel.

Reviewer: ```suggestion
<2> Specify `stable` or `stable-5.<y>` as the channel.
```

Author: I don't think this comma is necessary
